src/pykeen/datasets/__init__.py

```diff
@@ -78,7 +80,8 @@
 
 dataset_resolver = Resolver.from_entrypoint(group='pykeen.datasets', base=Dataset)
 if not dataset_resolver.lookup_dict:
-    raise RuntimeError(dedent('''\
+    raise RuntimeError(
```

Comment:
@sbonner0 could you please revert the stylistic changes in this PR

Reply:
Done - sorry about that! Something in the commit hooks changed the formatting.